system/modules/task/actions/edit.php

```diff
@@ -206,12 +209,13 @@ function edit_POST($w) {
     }
     
     $task->fill($_POST['edit']);
+    $task->rate = abs($task->rate) == 0 ? NULL : $task->rate;
```

Comment:
There you go, you learn something new everyday. I've never seen array_filter without a callback. Then the next problem with that is what if I want to actually include a space between two fields in a multiColForm? The only way to do this is by adding an empty field. If you're excluding fields from a form, the best way to do it is add it after the fact once you've created the base form. Checkout the opportunities edit action in our CRM module.

Regarding the abs use, if you use intval, then what you're comparing against will always be a integer.